corehq/apps/data_interfaces/views.py

```diff
@@ -609,37 +611,74 @@ def xform_management_job_poll(request, domain, download_id,
     return render(request, template, context)
 
 
-class BulkCaseReassignSatusView(DataInterfaceSection):
-    urlname = "bulk_case_reassign_status"
-    page_title = gettext_noop("Bulk Case Reassignment Status")
+class BulkCaseActionSatusView(DataInterfaceSection):
+    urlname = "bulk_case_action_status"
 
     @property
     def section_url(self):
-        return CaseReassignmentInterface.get_url(self.domain)
+        interface_class = CaseReassignmentInterface
+        if self.is_copy_action:
+            interface_class = CaseCopyInterface
+
+        return interface_class.get_url(self.domain)
+
+    @property
+    def page_title(self):
+        title = "Bulk Case Reassignment Status"
+        if self.is_copy_action:
+            title = "Bulk Case Copy Status"
+        return gettext_noop(title)
 
     def get(self, request, *args, **kwargs):
-        context = super(BulkCaseReassignSatusView, self).main_context
+        if self.is_copy_action:
+            action_text = "Copying"
+        else:
+            action_text = "Reassigning"
```

Comment:
I think you need to translate these, because on line 644 you're using them as if they're already translated.

Reply:
I'm always confused by string translations with variables. [This example](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#:~:text=This%20technique%20lets%20language%2Dspecific%20translations%20reorder%20the%20placeholder%20text.%20For%20example%2C%20an%20English%20translation%20may%20be%20%22Today%20is%20November%2026.%22%2C%20while%20a%20Spanish%20translation%20may%20be%20%22Hoy%20es%2026%20de%20noviembre.%22%20%E2%80%93%20with%20the%20month%20and%20the%20day%20placeholders%20swapped.) from the Django docs suggests that the string interpolation will be handled correctly without first translating the variable. I might be wrong?